package.json

```diff
@@ -36,6 +36,7 @@
     "add:shared": "npm install -w @wowarenalogs/shared",
     "add:parser": "npm install -w @wowarenalogs/parser",
     "add:tools": "npm install -w @wowarenalogs/tools",
+    "genPreload:app": "npm run genPreload --workspaces --if-present",
```

Comment:
nit: maybe "gen:app:preload" would be a bit more consistent. I was thinking of adding a one click graphql codegen script to the root level as well, so codegen might become a common action at some point